src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1334,19 +1339,95 @@ private static boolean hasParentModule(String sectionName) {
+                            "AvoidStarImport",
+                            "ConstantName",
+                            "DesignForExtension",
+                            "EmptyBlock",
+                            "EmptyForIteratorPad",
+                            "EmptyStatement",
+                            "EqualsHashCode",
+                            "FileLength",
+                            "FileTabCharacter",
+                            "FinalClass",
+                            "FinalParameters",
+                            "GenericWhitespace",
+                            "HiddenField",
+                            "HideUtilityClassConstructor",
+                            "IllegalImport",
+                            "IllegalInstantiation",
+                            "InnerAssignment",
+                            "InterfaceIsType",
+                            "JavadocMethod",
+                            "JavadocPackage",
+                            "JavadocStyle",
+                            "JavadocType",
+                            "JavadocVariable",
+                            "LeftCurly",
+                            "LineLength",
+                            "LocalFinalVariableName",
+                            "LocalVariableName",
+                            "MagicNumber",
+                            "MemberName",
+                            "MethodLength",
+                            "MethodName",
+                            "MethodParamPad",
+                            "MissingSwitchDefault",
+                            "ModifierOrder",
+                            "NeedBraces",
+                            "NewlineAtEndOfFile",
+                            "NoWhitespaceAfter",
+                            "NoWhitespaceBefore",
+                            "OperatorWrap",
+                            "PackageName",
+                            "ParameterName",
+                            "ParameterNumber",
+                            "ParenPad",
+                            "RedundantImport",
+                            "RedundantModifier",
+                            "RegexpSingleline",
+                            "RightCurly",
+                            "SimplifyBooleanExpression",
+                            "SimplifyBooleanReturn",
+                            "StaticVariableName",
+                            "TodoComment",
+                            "Translation",
+                            "TypecastParenPad",
+                            "TypeName",
+                            "UnusedImports",
+                            "UpperEll",
+                            "VisibilityModifier",
+                            "WhitespaceAfter",
+                            "WhitespaceAround"
+                    ));
+                    break;
+
+                default:
+                    Assert.fail("Missing modules list for style '" + styleName + "'");
```

Comment:
Please include file in failure message.

Reply:
Well, the `styleName` actually signals the file that is missing proper handling, while this is not an error in the style file, but in the test that was not prepared for a newly added style file.
What do you like more?
`Assert.fail("Missing modules list for style file '" + fileName + "'");`?

Reply:
You can have both if you wish, I just want the file name included so it is clear we got this error when processing this file. All errors in this test revolve around giving the fileName, so I would prefer to keep that so it is clear if any confusion arises.

Assert.fail("Missing modules list for " + styleName + " style in file '" + fileName + "'");

Reply:
`fileName` is always `${styleName}_style.xml`, so having both is pretty useless imho, that was also why I thought `styleName` is enough, especially as it is a "bug" in the test, not in the tested file.